meta-luneui/recipes-webos/pmloglib/pmloglib-private.bb

```diff
@@ -15,11 +15,11 @@ B = "${S}/build-private"
 EXTRA_OECMAKE += "-DBUILD_PRIVATE=ON"
 
 WEBOS_REPO_NAME = "pmloglib"
-SRC_URI = "${WEBOSOSE_GIT_REPO_COMPLETE}"
+SRC_URI = "${WEBOS_PORTS_GIT_REPO_COMPLETE};branch=webOS-ports/webOS-OSE"
```

Comment:
instead of setting ;branch=something after ${WEBOS_PORTS_GIT_REPO_COMPLETE} you should just set WEBOS_GIT_PARAM_BRANCH = "something"